packages/eslint-plugin/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@definitelytyped/eslint-plugin",
+  "version": "0.0.166-next.1",
+  "description": "ESLint rules for DefinitelyTyped",
+  "main": "./dist/index.js",
+  "publishConfig": {
+    "access": "public"
+  },
+  "contributors": [
+    "Nathan Shively-Sanders <nathansa@microsoft.com> (https://github.com/sandersn)",
+    "Andy Hanson <andy-ms@microsoft.com> (https://github.com/andy-ms)",
+    "Dan Vanderkam <danvdk@gmail.com> (https://github.com/danvk)",
+    "Remco Haszing <remcohaszing@gmail.com> (https://github.com/remcohaszing)"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/DefinitelyTyped-tools.git",
+    "directory": "packages/eslint-plugin"
+  },
+  "scripts": {
+    "build": "tsc -b .",
+    "test": "../../node_modules/.bin/jest --config ../../jest.config.js packages/dtslint"
+  },
+  "dependencies": {
+    "@definitelytyped/header-parser": "^0.0.166-next.1",
+    "@typescript-eslint/types": "^5.56.0",
+    "@typescript-eslint/utils": "^5.55.0"
+  },
+  "peerDependencies": {
+    "@typescript-eslint/eslint-plugin": "5",
+    "@typescript-eslint/parser": "5",
+    "eslint": "8",
+    "eslint-plugin-jsdoc": "44",
```

Comment:
These qualifiers look out of place; the other packages use carat notation. Probably they should match the monorepo?

Reply:
Other peer dependencies in this repo don’t match either format, but I changed these to use caret ranges.

Reply:
Dang! Mainly, I was just looking for the minimums here to look like the other packages, but I guess this monorepo isn't consistent?

Reply:
It’s not really inconsistent. Other packages only have one of these in their peer dependencies:

```jsonc
"typescript": ">= 3.0.0-dev || >= 3.1.0-dev || >= 3.2.0-dev || >= 3.3.0-dev || >= 3.4.0-dev || >= 3.5.0-dev || >= 3.6.0-dev || >= 3.7.0-dev || >= 3.8.0-dev || >= 3.9.0-dev || >= 4.0.0-dev || >=5.0.0-dev"
```

```jsonc
"typescript": "*"
```

So adding these new peer dependencies is basically a new concept for this monorepo.

Reply:
To be clear, I wasn't meaning only peer dependencies, I meant specifically that we may want to copy the version strings from this file: https://github.com/microsoft/DefinitelyTyped-tools/blob/4ff7996405e7b3a61ae47bbbbaca3f86f492b012/packages/dtslint/package.json#L29